src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -158,6 +166,19 @@ export class WalletOverviewPage {
     }
   }
 
+  getTransactionHistory(publicKey: string): Promise<void> {
+    return this.walletService.getTransactionHistory(this.currentWallet.publicKey).then(data => {
+      this.transactionsHistory = data;
+      if (this.transactionsHistory.length > 0) {
+        this.noTransactionHistoryVisibility = "hidden";
```

Comment:
I would say at least one of these variables is not needed since it is just the inverse of the other variable. E.g. if `noTransactionHistoryVisibility` is "hidden" then you can assume `transactionHistoryVisiblity` to be "shown".

Reply:
This variable is used for the [@visibilityChanged] in the html. By setting the variable to "shown" the opacity is set to 1 and if the variable is "hidden" the opacity is set to 0 and display is set to none. This is connected to the animations component at the top of the file